backend/ethereum_auth/email_verification.py

```
@@ -144,8 +143,8 @@ def decrypt_email(encrypted_email):
     return _fernet().decrypt(encrypted_email.encode()).decode()
 
 
-def _generate_raw_token():
-    return secrets.token_urlsafe(32)
+def _generate_verification_code():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_generate_verification_code`

Add return type annotation: `str`

(ANN202)